source/isaaclab/test/envs/test_scale_randomization.py

```diff
@@ -0,0 +1,345 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
```

Comment:
We can combine the two into a single test fixture with the event terms. Might make it easier to add new tests later instead of copying code for just function everytime :)